# One page review

### Short summary
A review paper which summarizes the recent findings in the study and
characterization of **CRISPR-Cas systems in Bacteria and Archaea**. Clustered
regularly interspaced short palindromic repeats (CRISPR) and Cas (CRISPR
associated proteins) is an *adaptive defense mechanism* present in this
primitive organisms, evolved in particular response to persistence stress from
biotic elements like viruses and transmissible genetic elements.

### Detailed summary
Microbes live and evolve in an environment with huge amount of diversity and
dynamic environment conditions that are stressful and fluctuating. These
stresses can be abiotic (e.g., non-optimal temperature or nutrient levels,
redox stress) and biotic (e.g., toxins, viruses, transmissible genetic
elements) in nature. The abundance of these factors, especially viruses is a
constant threat to the survival of these microbes. This, coupled with the fact
that viruses are prone to high rate of mutation and recombinations has led to
the development of sophisticated defense mechanism (multilayered,
fast-evolving) within the microbial host systems. These systems maintain
genetic integrity and co-evolve with the increasing variability and
sophistication of their predators; making up for one of the most variable
regions within the genome. However, they may occasionally allow exogenous DNA
uptake and conservation of genetic material advantageous for adaptation to the
environment.

Strategies like **prevention of adsoprtion, blocking of injection, abortive
infection, restriction-modification system (R-M)** and the use of
**sugar-nonspecific nucleases** are often used by hosts to target variety of
viral and non-viral attacks.

In this review, however, we examine one recently characterized and studied
(although it was discovered long back in 1987) called CRISPR-Cas system
(several studies pointed out similarity with RNAi mechanism in eukaryotic
systems). CRISPR, as the acronym stands, is a family of DNA repeats found in
most archaeal (~90%) and bacterial (~50%) genomes. Cas are CRISPR associated
genes that are located upstream of CRISPR loci and code for a family of
proteins that carry functional domains typical of nucleases, helicases,
polymerases, and polynucleotide-binding proteins.

CRISPR loci typically consists of several noncontiguous direct repeats
separated by stretches of variable sequences called spacers (which correspond
to segments of captured foreign DNA -- viral and plasmid). Although previous
studies were speculative of the role of these spacers, their origin from
extrachromosomal elements was established only in 2005 through three
independent studies.

These acquired spacer regions essentially act as a immunity repertoire or
memory for host cells and allow the effect of any external genetic material
containing the exact same spacer sequence, to be cleaved and neutralized using
the protein encoded by the *Cas* genes. A loose analogy to how a mammalian body
maintain a repertoire of antibodies (B cells actually) against the infections
it has encountered before.

**How does it happen?**

The repeat-spacer CRISPR array is first transcribed to give a pre-crRNA. The
full length pre-crRNA is subsequently processed into specific small RNA
molecules that correspond to a spacer flanked by two partial repeats. This
processing is achieved with the help of numerous enzymes and complexes
depending on the host system (e.g., CASCADE system in *E. Coli*, Cas6 in
*Pyrococcus*).

The processed crRNA, together with specific Cas proteins, form a CIRSPR
ribonucleoprotein (crRNP) complex that facilitates spacr base pairing to the
target or matching invasive nucleic acid. The crRNA acts as the guide to allow
for the specific base pairing between the exposed crRNA within the complex and
the corresponding **protospacer** on the foreign DNA. The complex then cleaves
the foreign DNA thereby neutralizing it.

The whole process thus can be summed up in three stages:

* Immunization, or spacer acquisition, involves the recognition and subsequent integration of spacers between two adjacent repeat units with the CRISPR locus. *Spacers appear to be integrate primarily at the leader end of the CRISPR loci.*
* CRISPR expression, a primary transcript, or a pre-CRISPR RNA (pre-crRNA) is
  transcribed from the CRISPR locus by RNA Polymerase. This pre-crRNA is then cleaved
  into small CRISPR RNAs (crRNAs) by specific endoribonucleases.
* Interference, or immunity, the crRNAs associated (or guide) a multiprotein
  complex to recognize and base-pair specific regions of incoming foreign DNA
  (or RNA) that have perfect (or almost perfect) complementarity.

**Minute details**

While the gist of the process has been described, the paper delves into much
detail by describing individual Cas proteins and different types of CRISPR-Cas
systems that have been identified depending on how they process the pre-crRNA
transcript. Besides, this system has also been observed to play critical role
in activity regulation (e.g., DNA repair, gene silencing) and development The
mechanism, involvement of complexes varies widely across specifies and any
reader is advised to refer the paper for those details.

Towards the end of the review, authors very neatly summarize the discussions
and pave way for future biotechnological implications of the CRISPR-Cas system.

* Strain Typing and Epidemiological Studies
* The rapid evolutionary dynamics of CRISPR sequences can provide important
  insights into the co-evolution of both host and phage populations and allow
  short, long term analysis by mathematical modeling.
* Development of specific/targeted strains.
* Recent application in the field of highly site specific **genome editing** by
  synthesizing custom guide RNAs.

### List of questions
1. If CRISPR-Cas system plays so important role, why did almost 50% of the
   bacterial population didn't acquire it (despite the fact that it is
   transferable horizontally)?
2. The role of the Cas genes need to be better characterized for the whole interplay
   to become more clear.
3. The actual role of PAMs, length of CRISPR loci and the factors which control these
   need to be explored further.
4. How this system could have evolved? What would it look like in even primitive
   and evolving systems? How did it change over time?

### Related work
1. Horvath, P. & Barrangou, R. CRISPR/Cas, the immune system of bacteria and archaea. Science 327, 167â€“170 (2010).

### Personal opinion
A comprehensive paper with updated facts and proper insight into the process.
The language is lucid and comprehensible but I found a few of the sections
towards the middle a bit repetitive, since that information was already
mentioned and explained in earlier section quite nicely.
